python/core/auto_generated/qgseditformconfig.sip.in

```diff
@@ -191,6 +191,19 @@ on the left hand side.
 Labeling on top leaves more horizontal space for the widget itself.
 %End
 
+    QString labelExpression( int idx ) const;
+%Docstring
+Returns the (possibly empty) expression for the label, to be evaluated in the form context.
+
+.. versionadded:: 3.14
+%End
+
+    void setLabelExpression( int idx, const QString &labelExpression );
```

Comment:
Could this be changed to field names instead of (supposedly) field index?

Reply:
of course yes, but I wanted to be consistent with the rest of the API, see : 
- bool labelOnTop( int idx ) const;
- bool readOnly( int idx ) const;

Reply:
Yeah I know... Ideally we would overload them with string methods too. And deprecate index usage.